package-lock.json

```diff
@@ -0,0 +1,11312 @@
+{
```

Comment:
We should stick with either `yarn.lock` or `package-lock.json` in this repo.

Reply:
Yes I didn't mean to commit package-lock.json.